src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -486,7 +489,11 @@ rct_window* window_scenery_open()
     gSceneryPlaceRotation = 0;
     gWindowSceneryPaintEnabled = 0; // repaint coloured scenery tool state
     gWindowSceneryEyedropperEnabled = false;
-    gWindowSceneryClusterEnabled = 0; // build cluster tool state
+    gWindowSceneryScatterEnabled = 0; // build cluster tool state
+    gWindowSceneryScatterWindow = nullptr;
+
+    gWindowSceneryScatterSize = 16;
+    gWindowSceneryScatterAmount = 35;
```

Comment:
Considering we've gone for a density selection, is this variable still used? (And if so, does this default value still make sense?)

Reply:
You were reviewing before I had implemented density. This variable has been replaced now.